packages/js/src/api/inbox-service.ts

```diff
@@ -64,25 +64,10 @@ export class InboxService {
       queryParams.append('archived', `${archived}`);
     }
 
-    const data = await this.#httpClient.get(`${INBOX_ROUTE}/notifications?${queryParams.toString()}`);
-
-    // TODO: fix this
-    return {
-      data,
-      hasMore: false,
-      filter: {},
-    };
+    return this.#httpClient.getFullResponse(`${INBOX_NOTIFICATIONS_ROUTE}?${queryParams.toString()}`);
```

Comment:
thankfully there is a `getFullResponse` function that returns the whole JSON object 🙏